source/index.html.erb

```diff
@@ -46,6 +46,26 @@ per_page: 4
   </div>
 </div>
 
+<div class="featured-tip">
+  <div class="container">
+    <% blog.articles.select{ |article| article.data.category == 'tips' }[0...1].each do |article| %>
+      <div class="left">
```

Comment:
I'd try and avoid class names like `left` and `right.` They are just too presentational. Maybe go more semantic here or think about abstracting a component you can reuse in the future.